command-line/keyboard-shortcuts.md

```diff
@@ -53,6 +53,18 @@ You don't have to use keyboard shortcuts for everything on your computer, but af
 
 - [Windows Keyboard Shortcuts Cheat Sheet](https://code.visualstudio.com/shortcuts/keyboard-shortcuts-windows.pdf)
 
+**General Mac Tips:**
+
+* **Two Finger Scrolling** - Slide two fingers up or down the trackpad to scroll.
+
+* **Pairing a Magic Mouse** - To pair the magic mouse with your Mac:
+
+    1. Turn it off.
+    2. Hold the 'mouse button' and while holding the button, turn it on.
+    3. Keep holding the buton till the green light starts blinking repeatedly (this puts it in the discovery mode).
```

Comment:
could you add :
through System Preferences > Bluetooth **or**  System Preferences > Mouse.

Reply:
Yes, done. Sorry I made a bunch of commits, won't happen next time.

Reply:
No worries!  But if you want, you can look up how to squash commits - it's pretty fun. Just remember to do it in a branch, and then if things go wrong, just delete that branch.

On a side note, remember to return to master, pull, and make a new branch every time you start on a new issue.  Most participants had to learn that the hard way this spring ;)